src/printer.js

```diff
@@ -1197,10 +1197,44 @@ function printClassPart(
   );
 }
 
+function printAttrs(path, options, print, inline = false) {
```

Comment:
Usually we're using objects instead of "non-speaking" boolean arguments in cases like this, such that the callsite looks like this:
```js
printAttrs(path, options, print, { inline: true });
```
See for example `getEncapsedQuotes`.

Reply:
I have changed it, definately makes code easier to read, be nice if there is was a quick way to parse that param to `path.call(...)`